SimPEG/Props.py

```diff
@@ -60,6 +60,7 @@ class Model(SphinxProp, properties.Array):
 
     class_info = 'a numpy array'
     _required = False
+    _shape = {('*',), ('*', '*')}
```

Comment:
@rowanc1: I added this to allow us to input anisotropic models

Reply:
cool!